### DISCUSSION TITLE
When will my benefits be activated

### DISCUSSION THREAD

**Discussion by ponsodie** [community/community#140090] [GitHub Education]
Good day I was approved on the 26th of September. My benefits are yet to be activated. Please help

**Comment by davevad93** [comment]
Hi @ponsodie ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ponsodie was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ponsodie** [reply]
Thank you for your time. I am excited for the opportunity. Happy weekend

**Reply by davevad93** [reply]
You're welcome @ponsodie , have a nice weekend you too! 👍
